xslt/Image.xsl

```diff
@@ -156,32 +111,32 @@
 
     <!-- /ImageDocument/Metadata/Information/Image/Dimensions/S/Scenes/Scene => /OME/Image/Image -->
     <xsl:template match="Scene">
-        <xsl:element name="Image">
+        <xsl:element name="ome:Image">
             <!-- Attributes -->
-            <xsl:attribute name="Id">
+            <xsl:attribute name="ID">
                 <xsl:text>Image:</xsl:text>
                 <xsl:value-of select="@Index"/>
             </xsl:attribute>
+            <xsl:attribute name="Name"/>
             <xsl:variable name="img" select="/ImageDocument/Metadata/Information/Image"/>
             <xsl:variable name="chs" select="/ImageDocument/Metadata/DisplaySetting/Channels"/>
-            <xsl:apply-templates select="$img/AcquisitionDateAndTime"/>
-            <xsl:apply-templates select="$img/SizeX"/>
-            <xsl:apply-templates select="$img/SizeY"/>
-            <xsl:apply-templates select="$img/SizeZ"/>
-            <xsl:apply-templates select="$img/SizeC"/>
-            <xsl:apply-templates select="/ImageDocument/Metadata/Experiment/AutoSave/DimensionsForSeparateFolders"/>    <!-- DimensionOrder -->
-            <xsl:apply-templates select="$chs/Channel[1]/PixelType"/>
-            <xsl:apply-templates select="$chs/Channel/BitCountRange[1]"/>
-            <xsl:apply-templates select="/ImageDocument/Metadata/Scaling/Items"/>
+<!--            <xsl:apply-templates select="$img/SizeX"/>-->
+<!--            <xsl:apply-templates select="$img/SizeY"/>-->
+<!--            <xsl:apply-templates select="$img/SizeZ"/>-->
+<!--            <xsl:apply-templates select="$img/SizeC"/>-->
+<!--            <xsl:apply-templates select="/ImageDocument/Metadata/Experiment/AutoSave/DimensionsForSeparateFolders"/>    &lt;!&ndash; DimensionOrder &ndash;&gt;-->
+<!--            <xsl:apply-templates select="$chs/Channel[1]/PixelType"/>-->
+<!--            <xsl:apply-templates select="$chs/Channel/BitCountRange[1]"/>-->
+<!--            <xsl:apply-templates select="/ImageDocument/Metadata/Scaling/Items"/>-->
```

Comment:
Remove commented out if not needed. BUT, not sure where we left off on this.... We definitely want Physical Pixel Sizes transferred over so maybe we need to add this back soon....

Reply:
I see a whole Pixels.xsl so I am hoping maybe these just got moved to their own file? Would be worth a check.

Reply:
For this 
1) check the schema to check it still makes sense
2) add it back
3) check it validates 
4) if not is it fixable? or remove it.